domain/dcsa_ovs_domain.yaml

```diff
@@ -0,0 +1,294 @@
+          
+          
+#        - type: object
+#          properties:
+#            eventID:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventID'
+#        - type: object
+#          properties:
+#            eventType:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventType'
+#        - type: object
+#          properties:
+#            eventDateTime:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventDateTime'
+#        - type: object
+#          properties:
+#            eventClassiefierCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_NOTIFICATION_DOMAIN/1.0.0#/components/schemas/eventClassifierCode'
+#        - type: object
+#          properties:
+#            EventTypeCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventTypeCode'
+#        - type: object
+#          properties:
+#            transportCallID:  
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/transportCallID'
+#        - type: object
+#          properties:
+#            locationType:
+#              $ref: '#/components/schemas/locationType'
+#        - type: object
+#          properties:
+#            locationID:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/locationID'
+#        - type: object
+#          properties:
+#            comment:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/eventComment'
+#        - type: object
+#          properties:
+#            delayReasonCode:
+#              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/delayReasonCode'
+      required:
+        - transportCallID
+        - timestampID
+        - eventClassiefierCode
+        - transportEventTypeCode
+        - locationType
+        - locationID
+        - eventDateTime
+
+############################
+#  Objects to be verified  #
+############################
+  
+    subscriptionID:
+      type: object
+      properties:
+        subscriptionID:
+          $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/subscriptionID'
+        
+
+
+    transportCallID:
+      type: object
+      nullable: false
+      properties:
+        transportCallID:
+          $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/transportCallID'
+    
+#########################
+#String variables below #
+#########################    
+ 
+ ################################################
+ ### Verfie locationType in the DCSA context ####
+ ################################################
```

Comment:
```suggestion
 #################################################
 ### Verfied locationType in the DCSA context ####
 #################################################
```

Reply:
This is a comment, as we need to discuss how to handle the Object location type, there is an other one in DCSA-DOmain with a different background 
Of course I correct the typo in verify